### DISCUSSION TITLE
关于advance_test

### DISCUSSION THREAD

**Discussion by Evens1sen** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#74] [Q&A]
在测试advance_test的时候，设置spiffy=True则会导致超时，若设定成False，可以通过测试。请问可能是什么地方有问题呢

**Comment by TANSixu** [comment]
Hi，设置spiffy=True则表示在在网络模拟器上运行（模拟器的名字的叫spiffy），会有模拟的网络路由，丢包，延迟等。如果运行超时可以自行延长max time，我们只有bonus会测时间，其他的时间都不重要的。
